packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -89,7 +99,9 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
 
   const fetchSmartTitle = useCallback(async () => {
     if (!fetchedSmartTitle) {
-      await refetch();
+      const [translateResult] = await fetchTranslations([post]);
+
+      client.setQueryData(key, translateResult?.value || post?.title);
```

Comment:
Fetch from kvasir directly, it will return the correct one depending on the clickbait shield setting